_posts/2019-03-06-welcome-new-hire-getting-started-in-frontend-at-rimdev.md

```diff
@@ -14,6 +14,10 @@ image_url: https://unsplash.com/photos/AqrD5QQ0ajM
 image_credit: "@ilumire"
 ---
 
+<div class="ui warning message">
+  This post is getting old! See an updated version <a href="">here</a>
```

Comment:
Why was href link taken out here?

Reply:
i changed go lang on the old post, undo took the link out as well - thanks! replacing.